config/_shared.json

```diff
@@ -1,3 +1,4 @@
 {
-	"discover_blog_id": 53424024
+	"discover_blog_id": 53424024,
+	"project": "project"
```

Comment:
It looks like it isn't possible to use `wpcom` or `wpcom.js`, but it works properly with `calypso` like Nikolay suggested.